.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 .nyc_output
 coverage
 dist
+/.eslintrc.js
```

Comment:
This file is not in the repo - if you've added this for your own benefit please remove!

Reply:
Okay, i got it. 
thanks.